### PR TITLE
Add Lina's branch with 1st test

### PR DESCRIPTION
**Pull request by angelinailchuk**

### CODE REVIEW

File: src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+public class PriceComparisonCartGridTest extends TestInit {
+    @Test
+    public void priceComparisonTest() {
+        HomePage homePage = new HomePage(driver);
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        driver.get("https://epicentrk.ua/");
+        homePage.popupWindow().click();
+        homePage.searchField().sendKeys("краска\n");
+        sleep(2);
```
Comment (StepaniukAndii) [inline src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java:17]: Why sleep()?

File: src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        for (int i = 0; i < 5; i++) {
+            js.executeScript("arguments[0].scrollIntoView();", homePage.buyBtns().get(i));
+            homePage.buyBtns().get(i).click();
+            sleep(1);
```
Comment (StepaniukAndii) [inline src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java:22]: Why sleep()?

File: src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java (hunk 1 above)
Comment (angelinailchuk) [reply src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java]: fixed

File: src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java (hunk 2 above)
Comment (angelinailchuk) [reply src/test/java/TestEpicentrByLynn/PriceComparisonCartGridTest.java]: It doesn't work without this sleep()

**Review by StepaniukAndii — APPROVED** [review]

**Review by StepaniukAndii — APPROVED** [review]

**Pull request merged**